extra-multimedia/feeluown-download/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME="feeluown-download"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for downloading music"
+PKGDEP="python-3 feeluown request"
+
+ABHOST=noarch
+ABTYPE=python
+NOPYTHON2=1
+
+PKGBREAK="fuocore"
```

Comment:
You should actually break *and* replace an older fuocore version, defining the last version of fuocore available from the repository.

See the [Debian Policy Manual](https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-in-other-packages).